HuntTheWumpus/HuntTheWumpus.cpp

```diff
@@ -201,66 +200,90 @@ int main()
 
     while (change.IsPlaying())
     {
+
         // Parse input.
         std::string input;
 
         std::cout << "Command? " << std::flush;
 
-        std::getline(std::cin, input);
-
-        // Split into strings.
-        const auto stringTokens = SplitString(input, " \t\n");
-
-        const auto command = ToLower(stringTokens[0]);
-
-        if (command[0] == 'm')
+        try
         {
-            if (stringTokens.size() < 2)
+            std::getline(std::cin, input);
+            // Split into strings.
+            const auto stringTokens = SplitString(input, " \t\n");
+
+            if (stringTokens.size() == 0)
             {
-                std::cout << "A Move command must be followed by the destination cave id.\n";
-                continue;
+                throw std::runtime_error("");
```

Comment:
Since we're in a while loop, you can also realize the control flow change you want with a continue operation like on line 227.  Think then about why a run-time error is appropriate here vs. just messaging and continuing.